docs/development.md

```diff
@@ -265,9 +265,30 @@ This instruction set assumes that the app identifier is `teammates-john`.
      `https://{version}-dot-teammates-john.appspot.com`, e.g `https://4-18-dot-teammates-john.appspot.com`.
 
 1. (Optional) You can run the tests against the deployed app.
+   * You need to setup `Gmail API` for the project as follows:
+   
+     **NOTE**
+     > No actual emails are sent on the local development server so delivery of emails is not tested, hence the Gmail API is
+     only required for testing the delivery of emails when testing against the staging server.
```

Comment:
Something like this (feel free to refine further)?

>**Explanatory notes (optional)**: 
>1. We need to set up the Gmail API as explained in this section because our test suite uses the Gmail API to access Gmail accounts used for testing (these accounts are specified in `test.properties`) to confirm that those accounts receive the expected emails from TEAMMATES. 
>2. This set up is needed _only_ when testing against the staging server because no actual emails are sent by the dev server and therefore delivery of emails is not tested when testing against the dev server.
>3. While it is possible to use the Gmail API of _any_ GAE project for this purpose, for convenience, the instructions below explain how to use the Gmail API of the TEAMMATES instance itself.